### PR TITLE
Updates: add process title to request data | Clone document name to include name of auth_viewer and unique 10-digit string

### PR DESCRIPTION
**Pull request by coderboy-exe**

### CODE REVIEW

File: app/helpers.py [hunk 1]
```diff
@@ -55,11 +65,13 @@ def cloning_document(document_id, auth_viewers, parent_id, process_id):
...
-            document_name = doc_name
-        else:
-            document_name = "~" + doc_name + "~"
+        for viewer in viewers:
+            doc_name = document["document_name"]
+            if has_tilde_characters(doc_name):
+                document_name = doc_name.replace('~', '')
+            else:
+                rand = randomize(10)
+                document_name = doc_name + "_clone_" + viewer + f"_{rand}"
```
Comment (WawinyEdwin) [inline app/helpers.py:74]: Keep the document very simple. doc_name + viewer, no need for _clone_ and random string

File: app/helpers.py [hunk 2]
```diff
@@ -44,6 +46,14 @@ def has_tilde_characters(string):
             return True
     return False
 
+def randomize(characters: int):
```
Comment (WawinyEdwin) [inline app/helpers.py:49]: not required

**Review by WawinyEdwin — CHANGES_REQUESTED** [review]

**Comment by coderboy-exe** [comment]
On it

**Comment by coderboy-exe** [comment]
Changes made

**Review by WawinyEdwin — APPROVED** [review]

**Pull request merged**